backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -51,14 +55,17 @@
 @RequiredArgsConstructor
 public class BulkUploadResultsToFhir {
 
-  private static final String ALPHABET_REGEX = "^[a-zA-Z]+$";
+  private static final String ALPHABET_REGEX = "^[a-zA-Z\\s]+$";
+  private static final String SNOMED_REGEX = "^[0-9]{9}$";
```

Comment:
looking at our SpecimenType table, SNOMEDs can be more than 9 digits, and that will fail the regex check.
```
Nasal aspirate specimen                429931000124105
Nasopharyngeal and oropharyngeal swab  433801000124107
Nasal washings                         433871000124101
```
https://www.simplereport.gov/metabase/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoyLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjIzfSwidHlwZSI6InF1ZXJ5In0sImRpc3BsYXkiOiJ0YWJsZSIsInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9fQ==

Reply:
good catch